lib/src/collections/observable_set.dart

```diff
@@ -116,3 +133,86 @@ class _DelegatingObservableSet<E> extends DelegatingSet<E>
     removeWhere((e) => !test(e));
   }
 }
+
+class _UnmodifiableEmptySet<E> extends IterableBase<E> implements Set<E> {
```

Comment:
oh I see... Dart Set does not have a const constructor, is that the issue? Yeah that seems to suggest maybe it isn't needed. Could `static const ObservableSet EMPTY` be `static final` ?